### PR TITLE
Penv 69 kube deploy support

### PR DESCRIPTION
**Pull request by nordicdyno**
add kubernetes deploy support for https://github.com/insolar/kube

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,5 +24,14 @@ WORKDIR /go/src/github.com/insolar/insolar
 RUN  set -eux; \
      groupadd -r insolar --gid=999; \
      useradd -r -g insolar --uid=999 --shell=/bin/bash insolar
-COPY --from=build /go/src/github.com/insolar/insolar/bin/insolar /go/src/github.com/insolar/insolar/bin/insolard /go/src/github.com/insolar/insolar/bin/keeperd /go/src/github.com/insolar/insolar/bin/pulsard /usr/local/bin/
+COPY --from=build /go/src/github.com/insolar/insolar/application/api/spec/api-exported.yaml /app/api-exported.yaml
```
Comment (yellowmegaman) [inline Dockerfile]: I would prefer to have it mounted as a volume from ConfigMap and not to be placed here at build stage. What do you think?

File: Dockerfile [hunk 2]
```diff
@@ -24,5 +24,14 @@ WORKDIR /go/src/github.com/insolar/insolar
...
 
+# add script and configs required for network bootstrap
+ADD scripts/kube/bootstrap/* /app/bootstrap/
```
Comment (yellowmegaman) [inline Dockerfile]: same

File: Dockerfile (hunk 1 above)
Comment (nordicdyno) [reply Dockerfile]: it's just binary asset which required and no reason to not put it outside image, probably best option to embed it inside binary (jira task?)

File: Dockerfile (hunk 2 above)
Comment (nordicdyno) [reply Dockerfile]: Agreed, but we can left it here if it's not critical and create task to fix it, or if it's too nasty I can move this to `kube` repo. Your thoughts?

Comment (yellowmegaman) [reply Dockerfile]: sounds reasonable

File: Dockerfile (hunk 1 above)
Comment (yellowmegaman) [reply Dockerfile]: yeah, sure

**Pull request merged**